### ISSUE TITLE
Simplify default pacoloco.yaml config

### ISSUE THREAD

**Issue by ericlay** [anatol/pacoloco#81]
I found it a little confusing to have the optional repositories included in the installed config file [`pacoloco.yaml`](https://github.com/anatol/pacoloco/blob/master/pacoloco.yaml.sample)
```
 quarry:
    url: http://pkgbuild.com/~anatolik/quarry/x86_64
 sublime:
    url: https://download.sublimetext.com/arch/stable/x86_64
```

It is not noted in the [configuration section](https://github.com/anatol/pacoloco#configure) that these are purely optional.  And possibly lead those who are unaware to have unused extra repositories configured. 

Arch wiki has a [sample config](https://wiki.archlinux.org/title/Pacman/Tips_and_tricks#Pacoloco_proxy_cache_server) that is a lot more bare bones.
```
port: 9129
repos:
  mycopy:
    urls:
      - http://mirror.lty.me/archlinux
      - http://mirrors.kernel.org/archlinux
```

IMO moving the sample config as is to `/usr/share` and documenting both the optional repositories as well the need to edit/move/recreate the file in `/etc` would be a more appropriate method.

At minimum, please signify that these are purely optional repositories and are not needed to function somewhere in the  [configuration section](https://github.com/anatol/pacoloco#configure)?

**Comment by Cody-Learner** [comment]
I'd like to add that I found the configuration confusing as well.

Under the 'repos:' section, how would I add my local AUR repo, as both a URL and file?

Under the 'archlinux-reflector:' section, I was somewhat confused because I use the reflector package and initially thought it was related to setting it up. 

Seems if I point a web browser to the pacoloco URL, http://192.168.2.139:9129/repo/archlinux/extra/os/x86_64, it shows the extra packages.  Then if point a browser to the pacoloco URL, http://192.168.2.139:9129/repo/archlinux/core/os/x86_64, it shows the extra packages rather than core. If I delete the x86_64 html file from the pacoloco cache, the x86_64 html file will be regenerated for which ever repo is searched in a browser the first time.

The issue with the web browser is not a big deal because who cares, but I did find it confusing while trying to set up pacoloco.

It would seem providing a bare minimal working config would make things less confusing, along with a man page for all the config file settings/options.

**Comment by anatol** [comment]
Thank you for filing this issue.

Historically `pacoloco.yaml.sample` is served as a showcase for the pacoloco features and included a lot of stuff that is rarely needed for a user. To some degree the sample file duplicates information already mentioned at README.

From other side the sample file is also used as a default config for the Arch package. And as you said it is a good idea to keep is simple and clean. I am fine with dropping all the confusing options from `pacoloco.yaml.sample`. Do you mind to send a PR for it?

**Comment by ericlay** [comment]
I can see the value in both sides as well. I have made a PR [here](https://github.com/anatol/pacoloco/pull/83). I think it's wise to keep options in the default config with the optional ones commented out. It serves as a good example, imo. I tried to keep things intact and just provide some slight guidance/explanation. Hope this is useful! :)

**Comment by anatol** [comment]
Closed with #83
